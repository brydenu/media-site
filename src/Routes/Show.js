import React from "react";
import MediaPage from "../Common/MediaPage";

/**
 * Show page with information about the show.
 */
export default function Show() {
    return <MediaPage mediaType="show" />;
}
