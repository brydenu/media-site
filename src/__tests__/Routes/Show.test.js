import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import Show from "../../Routes/Show";
import TestContext from "../../contextForTests";

it("renders without crashing", function () {
    render(
        <MemoryRouter>
            <TestContext>
                <Show />
            </TestContext>
        </MemoryRouter>
    );
});

it("matches snapshot", function () {
    const { asFragment } = render(
        <MemoryRouter>
            <TestContext>
                <Show />
            </TestContext>
        </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
});
