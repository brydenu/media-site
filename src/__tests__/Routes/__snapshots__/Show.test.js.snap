// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="back-btn"
  >
    Go Back
  </div>
  <div
    class="card-wrapper media-page main show"
  >
    <div
      class="card-content media-page main show"
    >
      <div
        class="card-header "
      >
        <h1>
          Loading
        </h1>
      </div>
      <div
        class="card-body media-page main show"
      >
        <div
          class="lds-default loading-animation"
        >
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
          <div />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
